net/init-metrics.sh

```diff
@@ -83,11 +83,16 @@ else
   query "CREATE DATABASE \"$netBasename\""
   query "ALTER RETENTION POLICY autogen ON \"$netBasename\" DURATION 7d"
   query "GRANT READ ON \"$netBasename\" TO \"ro\""
-  query "GRANT WRITE ON \"$netBasename\" TO \"scratch_writer\""
```

Comment:
I thought we are keeping this.

Reply:
it will use the username specified to the script, as it always should have (see next line in script).